lib/src/base-styles/_nx-btn.scss

```diff
@@ -24,14 +24,14 @@
   text-align: center;
   text-decoration: none;
 
-  &:hover {
+  &.nx-btn--secondary:hover {
     background-color: var(--nx-swatch-blue-90);
+    color: var(--nx-swatch-blue-30);
   }
 
   &:focus:not(:active) {
-    border-color: var(--nx-color-interactive-border-focus);
-    box-shadow: var(--nx-box-shadow-focus);
-    outline: 0;
+    outline-offset: -5px;
+    outline: 2px solid var(--nx-swatch-blue-30);
```

Comment:
@avnvbhatta I followed up with Jason today and they said to stick with `nx-swatch-blue-30` here - `nx-swatch-blue-40` didn't pass accessibility contrast when also in a hover state, which is why the outline was made slightly darker. They also caught that the text is supposed to darken as well on hover, so updated here: [a583477](https://github.com/sonatype/sonatype-react-shared-components/pull/983/commits/a58347772ba59f1dbbc711146a5f1702328ac96f)